src/main/java/com/poiji/bind/Poiji.java

```diff
@@ -359,11 +360,15 @@ private static HSSFPropertyStream deserializerPropertyStream(PoijiExcelType exce
      * @see Poiji#fromExcel(Sheet, Class, PoijiOptions)
      * @see Poiji#fromExcel(Sheet, Class)
      */
-    public static <T> void fromExcel(final Sheet sheet,
+    public static <T> void fromExcel(Sheet sheet,
```

Comment:
we need support all the `fromExcel` having the `Sheet` object

Reply:
Request support from more experienced..